src/metrics.rs

```diff
@@ -152,7 +152,8 @@ impl MetricsAggregator {
 }
 
 struct AggregatedSample {
-    // the histogram of message rates (msg/s) over all clients
+    // The histogram of message rates (msg/microsecond) over all clients.
```

Comment:
I am confused, should the comment say msg/million-seconds?

Reply:
Yes, I originally had this reversed and never fixed this place. Fixed now.